packages/botonic-eslint-config/index-ci.js

```diff
@@ -0,0 +1,28 @@
+// Eslint configuration for CI (slower but detects more issues)
```

Comment:
Does this mean that you might get issues that the editor is not telling you?

Reply:
yes, It's a compromise between agility on development machines (these options make IDEs and github hooks very slow) and test thoroughness